src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+/** @format */
+
+import { Link } from "react-router-dom";
+
+function Header() {
+	return (
+		<Link to="/">
+			<img
+				src="https://user-images.githubusercontent.com/23629340/40707029-cb2fce12-63ef-11e8-939c-f673ff3b965d.png"
+				alt="header image"
```

Comment:
Avoid `alts` with picture, image or img as this is a given already :) 

```suggestion
				alt="home page icon"
```